WordPress/src/main/java/org/wordpress/android/ui/stats/StatsCommentsFragment.java

```diff
@@ -82,7 +82,7 @@ protected void updateUI() {
             return;
         }
 
-        if (mDatamodels[1] != null) { // check if comment-followers is already here
+        if (mDatamodels[1] != null && !isErrorResponse(1)) { // check if comment-followers is already here
```

Comment:
This ensures that it's not a VolleyError, but there's no guarantee that it's a CommentFollowersModel. It would be safer to explicitly check `(mDatamodels[1] instanceof CommentFollowersModel)`.